lib/AcqList/hooks/useResultsSRStatus/useResultsSRStatus.js

```diff
@@ -0,0 +1,25 @@
+import {
+  useRef,
+  useEffect,
+} from 'react';
+import {
+  useIntl,
+} from 'react-intl';
+
+export const useResultsSRStatus = ({ hasFilters, resultsCount }) => {
+  const intl = useIntl();
+  const readerStatusRef = useRef();
+
+  useEffect(() => {
```

Comment:
It looks like if some filters are applied and `resultsCount` stays the same, the screen reader won't read it. It's ok?

Reply:
good point, don't think it should be silent